docs/topics/logging.txt

```diff
@@ -649,6 +649,27 @@ Python logging module.
         subclass the :class:`~django.utils.log.AdminEmailHandler` class and
         override this method.
 
+
+    The ``reporter_class`` argument of ``AdminEmailHandler`` is used to customize
+    the traceback ``ERROR`` sent to the site :setting:`ADMINS` when
+    :setting:`DEBUG` is set to ``False``. To set this value in your configuration,
+    include it in the handler definition for
+    ``django.utils.log.AdminEmailHandler``, like this:
+
+    .. code-block:: python
+
+        'handlers': {
+            'mail_admins': {
+                'level': 'ERROR',
+                'class': 'django.utils.log.AdminEmailHandler',
+                'include_html': True,
+                'reporter_class': 'somepackage.error_reporter.CustomErrorReporter'
+            }
+        },
+
+    Note that ``reporter_class`` should either inherit from ``django.views.debug.ExceptionReporter``
+    or implement all methods of ``django.views.debug.ExceptionReporter``
```

Comment:
Please add `.. versionadded::`.

Reply:
@felixxm I'm not sure about the version, Can you please mention which version should I add here?

Reply:
3.0

Reply:
done!